ui/client/http/HttpService.js

```diff
@@ -147,6 +147,10 @@ export default {
     return promiseWrap($.get(API_URL + '/processesDetails'))
   },
 
+  fetchProcessesComponents(componentName) {
+    return promiseWrap($.get(API_URL + '/processesComponents/' + componentName))
```

Comment:
will it work correctly for componentName (let's change to componentId?) where we need escaping e.g. with spaces?